demos/global-matrix-auth/README.md

```diff
@@ -12,8 +17,9 @@ jenkins:
         - "Overall/Read:anonymous"
         - "Overall/Administer:authenticated"
 ```
+Permissions must be defined **per line**, meaning each line must grant permission to only a single role, and only a single user or group of users.
 
-## sample-configuration (project based matrix)
+## sample-configuration (project-based matrix)
```

Comment:
```suggestion
## sample-configuration (project based matrix)
```

I would advice against that, because it breaks HTML anchors. AFAICT we have no English spelling requirements. I prefer using dashes too by the way (UK English?)